### PR TITLE
[16.0][ADD] stock_picking_kind

### PR DESCRIPTION
**Pull request by glitchov**
Computes the kind of picking based on source and destination locations of the picking

### CODE REVIEW

File: stock_picking_kind/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            ("supplier_return", "Supplier Return"),
+            ("supplier_in", "Supplier Reception"),
+            ("drop_shipping", "Drop Shipping"),
+            ("drop_shipping_return", "Drop Shipping Return"),
+        ],
+        compute="_compute_picking_kind",
+        store=True,
+    )
+
+    @api.depends("location_id", "location_dest_id")
```
Comment (rousseldenis) [inline stock_picking_kind/models/stock_picking.py:25]: @glitchov You should add usage

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by lmignon — APPROVED** [review]

**Comment by lmignon** [comment]
@rousseldenis Can you merge this one PLZ?

**Pull request merged**